nilearn/signal.py

```diff
@@ -417,6 +417,9 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         nilearn.image.clean_img
     """
 
+    if isinstance(low_pass, bool) or isinstance(high_pass, bool):
+        raise TypeError("high pass and low pass must be float or None")
```

Comment:
Best practice: always include the value provided by the user in the error message, something like:
```py
raise TypeError("High pass must be float or None but you provided high_pass='{0}'".format(high_pass))
```